magento/magento2
# Fix issue 21177 - Cart page cross-sell product add-to-cart button issue resolved

### Description (*)

Cart page cross-sell product add-to-cart button issue resolved
Magento 2.3-develop
### Fixed Issues (if relevant)

1.magento/magento2#21177: Cart Page cross sell product Add to cart button overlapping

### Manual testing scenarios (*)

1.Add any product to cart into frontend(Note : This product must have cross-sell product added from backend )
2.Check Cross-sell product into 640 * 767 screen.

### Contribution checklist (*)
